workflow/labels.md

```diff
@@ -52,3 +55,11 @@ Each review label has 3 steps:
 | Question         | To ask a question about something on the project                                                                                                                                                                     |
 | Good first issue | Lets potential contributors know that [the issue would be a good one to start with](https://help.github.com/articles/helping-new-contributors-find-your-project-with-labels/). For example, a typo or grammar error. |
 | Help wanted      | Alerts contributors that the assigned person (or if there is none, the team as a whole) would prefer outside help in fixing the problem.                                                                             |
+| Project Fuse ⚡️   | This issue is related to our Project Fuse, which aims to speed up the site for end users | 
+| Copy update      | This issue is about a copy update, no design, UX or code changes |
+| Bug 🐛           | This is a bug that needs fixing |
+| Enhancement ✨   | This is about enhancing functionality |
+| Maintenance 🔨   | A required fix to keep the software up to date |
```

Comment:
- Many of our repos already have labels “Type: Bug”, “Type: Enhancement”, and “Type: Maintenance”. If these labels were a good idea I think their current names would be better, since they’re grouped in the alphabetical list.
- However, in the repos I’ve seen, these labels are hardly ever used (and I don’t think it’s their length at fault). I think we stopped using them shortly before @nottrobin started the practices repo, on the grounds that using them wouldn’t improve anyone’s decision-making. Sometimes (quite often, actually!) implementing a roadmapped enhancement is higher priority than fixing an obscure bug. And dropping the labels would avoid inconsequential arguments about, for example, whether an accessibility issue counts as a bug or an enhancement.

Reply:
I agree. I'm unclear whether there's anyone who keenly wants to defend the existence of these distinctions, but if so, please speak up: I'd like to understand what the real-world utility might be to classifying things as "bug"s or "enhancement"s.

Reply:
@nottrobin we're using [release drafter](https://github.com/apps/release-drafter) to automatically generate release changelogs from PR commits and labels. Release drafter automatically categorises PRs by label in the draft changelog.

Reply:
> * Many of our repos already have labels “Type: Bug”, “Type: Enhancement”, and “Type: Maintenance”. If these labels were a good idea I think their current names would be better, since they’re grouped in the alphabetical list.

Prefixing these with "Type:" sounds fine, but we should keep the emojis.

Reply:
@squidsoup which ones are important for drafter?

Reply:
@squidsoup ah right. I didn't realise this was part of that mechanism. That seems like a good argument, for published packages. But it seems kind of a shame to pollute websites (the vast majority of our repos) with those labels without a reason.

Reply:
> @squidsoup which ones are important for drafter?

Totally customisable, but at the moment we're using "feature", "enhancement", "bug" and "maintenance"

Reply:
> @squidsoup ah right. I didn't realise this was part of that mechanism. That seems like a good argument, for published packages. But it seems kind of a shame to pollute websites (the vast majority of our repos) with those labels without a reason.

Sure, it's mostly useful if you're using release drafter and github releases - do we keep changelogs for our websites currently?

Reply:
No we don't have changelogs of websites. Since we're going for as close to a continuous-delivery system as we can, I don't think changelogs make any sense for websites.

Reply:
Updating following our discussion in dev catch-up - yes releases could be useful for websites if they provide an easy mechanism for us to indicate that an issue / PR had been released. We could look into this.